src/Components/OurWork/OurWork.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from 'react';
 import workData from './data';
+import { WorkData } from './data';
```

Comment:
```suggestion
import workData, { WorkData } from './data';
```

p.s. there is also `eslint`  rule that helps to enforce no import duplicates https://eslint.org/docs/latest/rules/no-duplicate-imports